src/app/containers/ArticleContainer.js

```diff
@@ -0,0 +1,54 @@
+import React, { Component } from 'react';
+import { connect } from 'react-redux';
+import { toast } from 'react-toastify';
+import ArticlePresentationComponent from '../components/ArticleComponent';
+import { postArticle } from '../../redux/actions/ArticleActions';
+
+export class ArticleContainer extends Component {
+  state = {
+    title: "",
+    description: "",
+    body: "",
+    tagList: ["tags"]
+  };
+
+  checkBlank = () => {
+    const { title, body, description } = this.state;
+    if (title === '' || body === '' || description === '') {
+      toast.dismiss();
+      toast.error("All fields must be filled", {
+        hideProgressBar: false,
+        autoClose: 3000
+      });
+    }
+  };
+
+  handleChange = (event) => {
+    this.setState({
+      [event.target.name]: event.target.value
+    });
+  };
+  handleSubmit = (event) => {
+    event.preventDefault();
+    const { title, body, description, tagList } = this.state;
+    const articleDetails = { title, body, description, tagList };
+    this.checkBlank();
+    const articleUrl = 'https://inviticus-staging.herokuapp.com/api/articles/';
+    const headers = {
+      'Content-Type': 'application/json',
+      'Authorization': 'Bearer eyJ0eXAiOiJKV1QiLCJhbGciOiJIUzI1NiJ9.eyJleHAiOjE1NTk4NDA5NDcsImlhdCI6MTU1OTgzOTE0Nywic3ViIjp7InVzZXJuYW1lIjoibWFyY3VzIiwiZW1haWwiOiJrYWJvaGFqZWFubWFya0BnbWFpbC5jb20iLCJwYXNzd29yZCI6IlNoaW5pZ2FtaUAxOTg3In19.uAUfiRfnkwBu6_ClZ-QZn2SnWT-4mVwC4X-MiK4hBoI'
+    };
+    this.props.postArticle(articleUrl, articleDetails, headers);
```

Comment:
Use redux dispatch method here instead of calling post articles directly

Reply:
I am using the dispatch method in my Actions in the file ArticleActions.js